src/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Editor } from 'grapesjs'
+import { KeymapsDialogManager } from './KeymapsDialogManager'
+
+const cmdKeymapsDialog = 'open-keymaps-dialog'
+
+export default (editor: Editor, opts = {}): void => {
+  const options = {
+    ...{
+      longPressKey: 'shift',
+      longPressDuration: 800,
+      shortcut: 'shift+k',
+      css: null
+    }, ...opts
+  }
+
+  const manager = new KeymapsDialogManager(editor, options)
+
+  // Command
+  editor.Commands.add(cmdKeymapsDialog, {
+    run(editor: Editor) {
+      manager.openDialog()
+    },
+    stop(editor: Editor) {
+      manager.closeDialog()
+    }
+  })
+
+  // Shortcut triggering the command
+  let isShortcutActive = false
```

Comment:
This variable is used inside the ``keyup`` event, so that the dialog doesn't close when releasing ``Shift`` if its opening was triggered by the shortcut (and not by the long press key).